### ISSUE TITLE
User Intruments/Roles

### ISSUE THREAD

**Issue by vwbusguy** [vwbusguy/band-schedule#6]
Allow users to select/add instruments or event roles that they are willing to do.  For example: Acoustic Guitar, Vocals, Soundboard, etc.

This should ideally happen on the User's Profile.  Perhaps displayed on the front page to make it more meaningful?

**Comment by ParadoxGuitarist** [comment]
It should happen on their Profile for them to enable which ones they can be signed up for, but it should also happen at the event level so people can see what's already singed up for.  (ie, they see that I'm signed up, but I can make it known that I'm running sound board that week and we still need guitarists, and keyboard players.)

**Comment by vwbusguy** [comment]
Did you see #7 referenced here?  Does it cover that concern?

**Comment by ParadoxGuitarist** [comment]
7 seemed to Indicate only the leader assigning roles.

**Comment by welkerc** [comment]
The musician/volunteer should be able to choose what they are willing to do for a specific week. If they can play keys, sing, and drum, but they are in a hand cast they'd only be able to sing for a time.

**Comment by ParadoxGuitarist** [comment]
I agree Welker, and there should be a way to sign up w/o assignment so the leader can indicate what they need from the volunteer.
